tests/baselines/reference/abstractPropertyInConstructor.errors.txt

```diff
@@ -90,4 +95,24 @@ tests/cases/compiler/abstractPropertyInConstructor.ts(39,22): error TS2715: Abst
             a.method2();
         }
     }
+    
+    abstract class Foo {
+        abstract x: string;
+        abstract y: string;
+    
+        constructor() {
+            let { x, y: y1 } = this;
+                ~~~~~~~~~~~~
+!!! error TS2715: Abstract property 'x' in class 'Foo' cannot be accessed in the constructor.
+                ~~~~~~~~~~~~
+!!! error TS2715: Abstract property 'y' in class 'Foo' cannot be accessed in the constructor.
+            ({ x, y: y1, "y": y1 } = this);

```

Comment:
just to check, can you include one non-erroneous access for both examples, as well as a `let self = this` declaration?